tests/esm-import-assertions/importJson.ts

```diff
@@ -0,0 +1,14 @@
+import carData from './car.json' assert { type: 'json' };
```

Comment:
Should we add a test that omitting the assertion causes the import to fail?  Is that a meaningful test or no?  Wondering if in the future we might accidentally introduce a bug that allows importing JSON modules *without* the necessary assertion.

Reply:
If you think it's a good test then definitely!

I would expect node itself to do the assertion and prevent parsing and running anything with `assert { type: 'json' }`as JS, but you know better which way ts-node might do it differently that might leak through node's checks.

Reply:
Ok, if it's not too much work, let's add the test.  I bet we can add something as simple as this:
```
await import('./car.json')
.then(
  () => {throw 'should have thrown'},
  () => {/* error is expected */}
)
```

That should do the trick, right?

Reply:
Test has been added.